gateway/ln-gateway/src/bin/gatewayd.rs

```diff
@@ -47,6 +49,11 @@ pub struct GatewayOpts {
     /// Gateway webserver authentication password
     #[arg(long = "password", env = "FM_GATEWAY_PASSWORD")]
     pub password: String,
+
+    /// Configured gateway routing fees
+    /// Format: <base_msat>,<proportional_millionths>
```

Comment:
Nit: For me I know I would be confused by this format, would probably just prefer separate parameters